### ISSUE TITLE
sendNotifications: crash

### ISSUE THREAD

**Issue by Reggian** [realm/realm-cocoa#3119]
Greetings!

Today I have encountered a crash out of the blue:

```
-[__NSTaggedDate sendNotifications:]: unrecognized selector sent to instance 0xe000000014217fa6
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[__NSTaggedDate sendNotifications:]: unrecognized selector sent to instance 0xe000000014217fa6'
*** First throw call stack:
(0x182495900 0x181b03f80 0x18249c61c 0x1824995b8 0x18239d68c 0x100504330 0x1005338a4 0x100533054 0x100532ff4 0x100526e04 0x1003e740c 0x1003e73e4 0x18244cefc 0x18244c990 0x18244a690 0x182379680 0x183888088 0x1871f0d90 0x1000981a4 0x181f1a8b8)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

<img width="1075" alt="line" src="https://cloud.githubusercontent.com/assets/1903894/12445031/4f1cc2c8-bf62-11e5-9fee-99c6cc666d4e.png">
<img width="1236" alt="trace" src="https://cloud.githubusercontent.com/assets/1903894/12445030/4f1c197c-bf62-11e5-8aed-6b9a5b1dc121.png">

```
Running on: iPhone 6s, iOS 9.2

ProductName:	Mac OS X
ProductVersion:	10.11.2
BuildVersion:	15C50

/Applications/Xcode.app/Contents/Developer
Xcode 7.2
Build version 7C68

/usr/local/bin/pod
0.39.0
Realm (0.97.0)

/bin/bash
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin15)

/usr/local/bin/carthage
0.10.0
(not in use here)

/usr/bin/git
git version 2.5.4 (Apple Git-61)
```

**Comment by tgoyne** [comment]
This was probably fixed by https://github.com/realm/realm-cocoa/pull/3078, so try pointing CocoaPods at master rather than 0.97.0.

**Comment by Reggian** [comment]
Thank you.
Will do and report if anything similar reoccurs.

**Comment by kishikawakatsumi** [comment]
I believe this issue is duplicate of #3013 and #3035. They are fixed at #3078. I'm going to close this issue. If you encounter the further problem, please feel free to reopen this issue or create a new one.
